app/src/main/java/com/h/pixeldroid/PostCreationActivity.kt

```diff
@@ -116,62 +127,139 @@ class PostCreationActivity : AppCompatActivity() {
         val textField = findViewById<TextInputEditText>(R.id.new_post_description_input_field)
         val content = textField.text.toString()
         if (content.length > maxLength) {
-            // error, too much characters
-            textField.error = "Description must contain $maxLength characters at most."
+            // error, too many characters
+            textField.error = "Description can contain at most $maxLength characters."
```

Comment:
strings.xml this

Reply:
je voulais pas re-faire le travail de la PR de @mjaillot

Reply:
pas sûr que ça marche, mais tu peux essayer de stocker "Description can contain at most %d characters." dans strings.xml et ensuite faire str.format(maxLength), ça peut être stylé

Reply:
C'est à peu près comme ça qu'il faut faire, j'avais fait ça pour les notifications. Mais je pense c'est mieux fait dans la PR de @mjaillot